config/install/node.type.su_basic_page.yml

```diff
@@ -0,0 +1,18 @@
+uuid: 1309a148-dad2-48ca-84bf-eb8f9dedc91a
+langcode: en
+status: true
+dependencies:
+  module:
+    - menu_ui
+third_party_settings:
+  menu_ui:
+    available_menus:
+      - main
+    parent: 'main:'
+name: 'Basic Page'
+type: su_basic_page
```

Comment:
Please have a review of the naming conventions document: https://docs.google.com/spreadsheets/d/1ATP3boyFio_s2BYjvzBoOt29p7aQrQV3HB-kZOZwp0g/edit#gid=0

@boznik please also have a review of the naming conventions document.

Reply:
got it, should be stanford_basic_page

Reply:
can we review the naming convention on fields more. field config & field storage config are namespaced `field.` already. Is there a need to have `field_s` before every field

Reply:
good point

Reply:
You both have been assigned a review of that document. Please review and leave comments.

Reply:
i left 2 comments, one adjusting the machine name and another agreeing that it would be nice to not add another field_ to the field config and storage name.

Reply:
We have the ability to customize the prefix so I would not expect us to create anything such as `field_field_s_thing` but rather just `field_s_thing`

Reply:
Where did you leave your comments?

Reply:
above

Reply:
https://github.com/SU-SWS/stanford_profile/pull/2/files#diff-c508ab451427faf121fbf79be6af723eR1
this changes `field_` to `su_` because a field config and storage has `field.field.[entity_type]` and `field.storage.[entity_type]` do we really need more indication its a field? when the name gets too long, the revision table is a hashed string

Reply:
I agree with those points, and the hashed strings are annoying but there are benefits to being the same as everyone else. We should discuss.

Reply:
discussed .`su_` it is.